client/src/main/java/org/apache/uniffle/client/impl/ShuffleReadClientImpl.java

```diff
@@ -82,6 +83,7 @@ public ShuffleReadClientImpl(
     this.blockIdBitmap = blockIdBitmap;
     this.taskIdBitmap = taskIdBitmap;
     this.idHelper = idHelper;
+    this.shuffleServerInfoList = shuffleServerInfoList;
```

Comment:
The size of shuffleServerInfoList will always be one because we use MultiReplicaReadHandler.

Reply:
No, `shuffleServerInfoList` has not be modified. You can check it again.

Reply:
OK, my mistake.